lua/lvim/core/luasnip.lua

```diff
@@ -0,0 +1,162 @@
+local M = {}
+
+-- see luasnip/util/types.lua
+local types = {
+  textNode = 1,
+  insertNode = 2,
+  functionNode = 3,
+  snippetNode = 4,
+  choiceNode = 5,
+  dynamicNode = 6,
+  snippet = 7,
+  exitNode = 8,
+  restoreNode = 9,
+  node_types = { 1, 2, 3, 4, 5, 6, 7, 8, 9 },
+}
+
+function M.config()
+  lvim.builtin.luasnip = {
```

Comment:
Put the builtin opts inside `opts`

Reply:
Also, add enabled condition to the config table and plugins table. Use it for all three snip plugins I think.